lunatrace/bsl/backend/src/workers/queue-processors/process-luna-trace-sqs-message.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright by LunaSec (owned by Refinery Labs, Inc)
+ *
+ * Licensed under the Business Source License v1.1
+ * (the "License"); you may not use this file except in compliance with the
+ * License. You may obtain a copy of the License at
+ *
+ * https://github.com/lunasec-io/lunasec/blob/master/licenses/BSL-LunaTrace.txt
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+import { createGithubWebhookInterceptor } from '../../github/webhooks';
+import { LunaTraceSqsMessage, MessageTypeToActivityLookup } from '../../types/sqs';
+import { MaybeErrorVoid } from '../../types/util';
+import { log } from '../../utils/log';
+import { processGithubWebhookActivity } from '../activities/process-github-webhook-activity';
+import { snapshotRepositoryActivity } from '../activities/snapshot-repository-activity';
+
+export async function createMessageTypeToActivityLookup(): Promise<MessageTypeToActivityLookup> {
```

Comment:
ok so activities = handlers

Reply:
If I wrote this, this would be `createQueueHandlers`.  This isn't a dynamically generated lookup table, its a flat object that has the instantiated queue handlers in it

Reply:
I am specifically trying to avoid "handler" anywhere in the queue code because that term is very overloaded. I chose "activity" because this is the term that is used in Temporal to define a "single, well defined action" https://docs.temporal.io/concepts/what-is-an-activity/. If/when we chose to use temporal, all of the queue code will go away, but the activities, as they are defined now, will not need to be changed at all.